src/Libraries/DSOffice/CSVNodes.cs

```diff
@@ -21,30 +20,20 @@ public static partial class Data
         /// <search>write,text,file</search>
         public static void ExportCSV(string filePath, object[][] data)
         {
-            try
```

Comment:
why remove the try catch here?

Reply:
reverted the rebased changes, it should be good now @reddyashish !